pkgs/top-level/beam-packages.nix

```
@@ -17,7 +17,7 @@ in
 {
   beamLib = callPackage ../development/beam-modules/lib.nix { };
 
-  latestVersion = "erlang_25";
+  latestVersion = "erlang_27";
```

```suggestion
  latestVersion = "erlang_27";
```
This should happen, but it's necessary to test/adapt the downstream packages.